tests/test_exceptions.py

```diff
@@ -50,13 +50,23 @@ def test_python_call_in_catch():
     assert d["good"] is True
 
 
+if hasattr(pytest, "PytestUnraisableExceptionWarning"):  # Python >= 3.8 and pytest >= 6
+    ignore_pytest_unraisable_warning = pytest.mark.filterwarnings(
+        "ignore::pytest.PytestUnraisableExceptionWarning"
+    )
+else:
+    ignore_pytest_unraisable_warning = lambda f: f  # NOQA: E731
```

Comment:
Yes, I could not put a lambda, and put 2 lines `def ignore_pytest_unraisable_warning(f):` and `    return f`, but then `black` suddenly adds an empty line after `else:`, so that's just as ugly.